components/auth/signIn.tsx

```
@@ -1,139 +1,140 @@
-      <div className="mt-6 space-y-4">
-        <GoogleSignIn text="Sign In" />
-        <DiscordSignIn text="Sign In" />
-      </div>
-    </>
-  );
+				<div className="sm:flex sm:items-center sm:gap-4">
+					<button
+						className={`inline-block w-full sm:w-auto shrink-0 rounded-md border px-12 py-3 text-sm font-medium transition focus:outline-hidden focus:ring-3 text-black bg-accent hover:border-neon-green hover:bg-neon-green hover:bg-neon-green dark:hover:text-neon-green dark:border-neon-green dark:bg-neon-green dark:hover:bg-transparent`}
+					>
+						{isLoading ? (
+							<svg className="animate-spin h-5 w-5 mx-auto" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)